src/notification.ts

```diff
@@ -0,0 +1,272 @@
+import ApiClient from './api_client';
+import { RequestPromise } from "request-promise";
+
+export default class NotifyClient {
+
+  apiClient: ApiClient;
+
+  constructor(params: {apiKeyId: string, baseUrl?: string}) {
+    this.apiClient = new ApiClient(params);
+  }
+
+
+    /**
+     * Usage:
+     *
+     * notifyClient = new NotifyClient({ apiKeyId, baseUrl });
```

Comment:
Don't think we need this example here. Will be in the docs. Plus gets out of date.